client/types/callbacks.go

```diff
@@ -129,7 +129,137 @@ type Callbacks interface {
 	// syncer can be used to initiate syncing the package from the server.
 	OnAgentPackageAvailable(addons *protobufs.AgentPackageAvailable, syncer AgentPackageSyncer) error
 
+	// OnCommand is called when the server requests that the connected agent perform a command.
+	OnCommand(command *protobufs.ServerToAgentCommand) error
+
 	// For all methods that accept a context parameter the caller may cancel the
 	// context if processing takes too long. In that case the method should return
 	// as soon as possible with an error.
 }
+
+type CallbacksStruct struct {
```

Comment:
I'm not sure if moving `callbacks.go` to types is a good fit. I think this directory contained interface only until now

Reply:
I use it in `client/internal/receiver_test.go` and needed to move it out of client to avoid an import cycle.

This struct is just a convenience that has no dependencies outside of types, protobufs, and context. There is no implementation logic.

Reply:
I think moving is fine. IIRC, the `types` package was created to break the dependency cycle. Alternatively we could move `CallbacksStruct` to its own package, but I don't think it is worth creating a new package just for one struct.

Reply:
Thanks for the explanation